addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -963,6 +964,24 @@ options.Class.include({
             reloadEditor: true,
         });
     },
+    /**
+     * This handler prevents reloading the page twice with a `request_save`
+     * event when a widget is already going to handle reloading the page.
+     *
+     * @param {OdooEvnet} ev
+     * @returns
+     */
+    _onSaveRequestDoubleReload(ev) {
```

Comment:
I don't like event handler being named with what they do. IMO `_onSaveRequest` would be better, then inside the handler you actually check for a double reload. If you want to later add stuff inside that method, it's better but also, this method currently looks like a method which is called when a double reload is asked.